### DISCUSSION TITLE
72 Hours

### DISCUSSION THREAD

**Discussion by Symbai** [dotnet/wpf#6310] [General]
https://github.com/dotnet/wpf/blob/main/roadmap.md#triaging-github-issues Since weekend (monday) PRs are not reviewed and issues are not labeled/commented. Are the mentioned 72 hours still up to date?

**Comment by gurpreet-wpf** [comment]
**About non-code PRs**
We are committed to a 72 hour (3 working days) turn around on triaging and responding to new non-code PRs filed on GitHub. These are PRs for documentations bugs, documentation enhancements or a general question not related to source code issues/bugs.

@Symbai as mentioned in our roadmap, this 72-hour commitment is for non-code PRs. Meanwhile, we are actively working on addressing issues and reviewing code PRs from our community as per our resources.

**Comment by Symbai** [comment]
![image](https://user-images.githubusercontent.com/14368203/160121660-deaf0414-8387-4673-aef8-3a3f8d383fb0.png)

If you quote then you should do it right. The 72 hours are also for addressing issues. From Monday to Thursday the new issues haven't even received a label. Pull requests hasn't been reviewed. Again, are the 72 hours still up to date?
